prefetch.bs

```diff
@@ -100,6 +100,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: determine the origin; url: determining-the-origin
       text: navigable container; url: navigable-container
       text: parent; url: nav-parent
+      text: top-level traversable set; url: top-level-traversable-set
```

Comment:
You can delete this as long as you change `[=top-level traversable set=] ` to `[=user agent/top-level traversable set=]`.

(You can tell by looking at https://dontcallmedom.github.io/webdex/t.html#top-level%20traversable%20set%40%40user%20agent%40dfn which says `(concept for user agent)`.)

Reply:
Ah I keep forgetting to check this github before making additions to include external specs. Will try to keep this in mind, thanks. Also made the corresponding change in prerendering.bs.